easypost/report.py

```diff
@@ -8,7 +8,22 @@ class Report(AllResource, CreateResource):
     def create(cls, api_key=None, **params):
         """Create a report."""
         requestor = Requestor(api_key)
-        url = "%s/%s" % (cls.class_url(), params["type"])
+        class_url = cls.class_url()
+        type = params["type"]
```

Comment:
Please see my comment above about intermediate variables - the same applies for this `type` variable here.